.gitignore

```
@@ -442,3 +442,5 @@ nl-kat-*
 # rpki cache
 /boefjes/boefjes/plugins/kat_rpki/rpki.json
 /boefjes/boefjes/plugins/kat_rpki/rpki-meta.json
+
+bench*.pstat
```

I think in general we can skip profilers `.pstat` files
```suggestion
*.pstat
```